client/src/views/Models/Model.vue

```diff
@@ -376,8 +377,6 @@
     }
 
     onNodeClick (node: GraphNodeInterface): void {
-      // Select which tab should be open first, then open the drilldown.
-      this.drilldownActiveTabId = 'metadata';
```

Comment:
Why removing it? Don't we want the panel to always start with the _Metadata_ panel?

Reply:
I thought we already initialized it above but you're right, I'll put it back.

Reply:
It is initialized at the beginning. But if you remove this it will not open on the metadata panel each time you click on a node.